ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -33,7 +33,6 @@ import { SubmitAnnouncementService } from '../../services/submitAnnouncement.ser
 import { AddressVerificationWarnings } from '../../models/addressVerificationWarnings';
 
 interface ComponentState {
-  inputAddress: Address | null;
   isReadOnly: boolean;
```

Comment:
Keeping a copy of inputAddress in this component was not helpful. We are going to try to find the default or temporary address and set it immediately on the child input component where it is actually used.